site/assets/js/modal-controller.js

```diff
@@ -60,6 +67,8 @@ var modalController = (function () {
         selectedStateIdx = 56;
       }
     }
+
+    g.selectAll("path").filter(function (d) { return d.id === selectedStateIdx;}).classed("active", true)
```

Comment:
nice moves. looks like Array.Prototype.Filter is supported back to IE9. do we trust it? worth bringing in like underscore?